.eslintrc.js

```diff
@@ -44,6 +44,12 @@ module.exports = {
             "error",
             "never"
         ],
-        "no-var": "error"
+        "no-var": "error",
+        // part of Node Style-guide but ignored
+        "max-len": 0,
+        "max-statements": 0,
+        "object-curly-spacing": 0,
+        "brace-style": 0,
```

Comment:
We use the "1tbs" brace style. 

I recommend changing this to  ["error", "1tbs", { "allowSingleLine": true }]

Reply:
:+1: 

it wasn't my intention to change the rules at all in fact, i just want to replicate the same with eslint.